statistics-production/api-data-standards.qmd

```diff
@@ -0,0 +1,225 @@
+| Total              | 2             | 50           | 100            |                  
+
+: Pupil counts and percentages in tidy format
+
+:::
+
+### Example 2 - Metrics with hierarchical filters
+
+#### Example of bad practice
+
+The following would not be accepted for publication via the API.
+
+::: {.table-responsive}
+
+| attendance |overall_absence | authorised_absence | unauthorised_absence | attendance_percent |overall_absence_percent | authorised_absence_percent | unauthorised_absence_percent |
+|------------|----------------|--------------------|----------------------|------------|----------------|-------------------|----------------------|
+| 180        | 20             | 12                 | 8                    | 90         | 10             | 6                 | 4                    | 
+
+: Attendance statistics in non-tidy format
+
+:::
+
+#### Example of good practice
+
+The following would be accepted for publication via the API. In this case, creating a hierarchical filter combination allows a clear representation of the data.
+
+::: {.table-responsive}
+
+| attendance_status  | attendance_type      | session_count  | session_percent |
+|--------------------|----------------------|----------------|-----------------|
+| Attendance         | Total                | 180            | 90              |
+| Absence            | Total                | 20             | 10              |
+| Absence            | Authorised absence   | 12             | 6               |
+| Absence            | Unauthorised absence | 8              | 4               |         
+
+: Attendance statistics in tidy format with hierarchical filters
+
+:::
+
+### Example 3 - Metrics with non-compatible filters
+
+#### Example of bad practice
+
+The following would not be accepted for publication via the API.
+
+::: {.table-responsive}
+
+| pupil_count_grade9to5 | pupil_count_grade9to4 | pupil_count_grade9to1 | pupil_percent_grade9to5 | pupil_percent_grade9to4 | pupil_percent_grade9to1 | progress8_score_male | progress8_score_female | progress8_score |attainment8_score_male | attainment8_score_female | attainment8_score |
+|------------------------|------------------------|------------------------|--------------------------|--------------------------|-------------------------|----------------------|------------------------|-----------------|----------------------|------------------------|-----------------|
+| 30                     | 40                     | 50                     | 60                       | 80                       | 100                      | 0.2                  | 0.21                   | 0.21            |0.09                  | 0.08                   | 0.10            |
+
+: Attainment grade rates and scores in non-tidy format
+
+:::
+
+#### Example of good practice
+
+The following would be accepted for publication via the API. In this case, splitting the data into separate data files is required in order to create tidy data structures.
```

Comment:
I've had a play with this a little as I still instinctively see the two tables and try to put as one. I think we're right on splitting them, but in the description we need to make it really clear why we are. E.g. that because they're different incompatible measurements they'd otherwise end up with something like this table on the right

![image](https://github.com/user-attachments/assets/544dd38b-ef60-4a29-b8d1-402cdc4615c5)

Reply:
Have added something like this under the example of **bad** data bit.

Reply:
Yeah I think that works well - think that example helps clarify it more